tests/unit/samplegen/test_integration.py

```diff
@@ -133,5 +134,129 @@ def test_generate_sample_rpc_not_found():
         {"Mollusc": DummyService({})}, DummyNaming("pkg_name"))
     sample = {"service": "Mollusc", "rpc": "Classify"}
 
-    with pytest.raises(samplegen.RpcMethodNotFound):
-        samplegen.generate_sample(sample, True, env, schema)
+    with pytest.raises(types.RpcMethodNotFound):
+        list(samplegen.generate_sample(sample, True, env, schema))
+
+
+def test_generate_sample_config_fpaths(fs):
+    expected_path = 'cfgs/sample_config.yaml'
+    fs.create_file(
+        expected_path,
+        contents=dedent(
+            '''
+            ---
+            type: com.google.api.codegen.SampleConfigProto
+            config_schema_version: 1.2.0
+            samples:
+            - service: google.cloud.language.v1.LanguageService
+            '''
+        )
+    )
+    actual_paths = list(gapic_utils.generate_all_sample_fpaths(expected_path))
+
+    assert actual_paths == [expected_path]
+
+
+def test_generate_sample_config_fpaths_directories(fs):
+    good_contents = dedent(
+        '''
+        ---
+        type: com.google.api.codegen.SampleConfigProto
+        config_schema_version: 1.2.0
+        samples:
+        - service: google.cloud.language.v1.LanguageService
+        '''
+    )
+    bad_contents = 'bad contents'
+    # We need some invalid configs in the directory as well to verify that
```

Comment:
nit: I'd move the comment above the `bad_contents` assignment

Reply:
Done.